src/test/java/io/cdap/plugin/servicenow/source/ServiceNowSourceConfigHelper.java

```diff
@@ -43,6 +43,7 @@ public static ConfigBuilder newConfigBuilder() {
     private String applicationName = "";
     private String tableNameField = "tablename";
     private String tableName = "tablename";
+    private String tableNames = "tablesnames";
```

Comment:
Why do we have 2 fields - tablename and tablenames. Seems confusing

Reply:
`tableName` is to store the table name entered by the user when the mode of  query is set to Table Mode in Service Now Batch Source Plugin
`tableNames` is to store all the table names entered by the user in Service Now Multi-Source Plugin

Reply:
I see.. please add a comment indicating what these fields are used for

Reply:
Comment added. [c5d6fbb](https://github.com/data-integrations/servicenow-plugins/pull/8/commits/c5d6fbb1b2d49b43e78aadd471064b92574c868e)